firmware/components/micropython/esp32/modam2320.c

```diff
@@ -0,0 +1,52 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <string.h>
+
+#include "py/mperrno.h"
+#include "py/mphal.h"
+#include "py/runtime.h"
+
+#include <driver_am2320.h>
+
+#ifdef CONFIG_DRIVER_AM2320_ENABLE
+
+// This is the function which will be called from Python as am2320.get_temperature().
+STATIC mp_obj_t am2320_get_temperature() {
+    
+    float temperature = 0;
+    driver_am2320_get_temperature(&temperature);
```

Comment:
What's the reason you're working with pointers instead of return values here?

Reply:
Good point that was left from some earlier code, changed to return value